apps/upgrade-single-node/create.py

```diff
@@ -0,0 +1,56 @@
+import json
+import time
+
+import weaviate
+import weaviate.classes as wvc
+
+sphere_file = "sphere.10k.jsonl"
+
+sizes = [10, 100, 1000, 10000, 10, 10, 10, 10, 50]
+
+
+def _import():
+    client = weaviate.connect_to_local()
```

Comment:
with weaviate.connect_to_local() as client:
   ...rest of code....
   
then the client will be closed automatically at the end of the contextmanager